shared/tracker/index.js

```diff
@@ -58,18 +59,24 @@ class Tracker extends Component {
 
     const renderChangedTitle = this.props.trackerMessage
     const failedProofsNotFollowingText = `Some of ${this.props.username}'s proofs couldn't be verified. Track the working proofs?`
+    const currentlyFollowing = flags.tracker2 ? this.props.currentlyFollowing : !!this.props.lastTrack
+
+    const changed = !this.props.proofs.every(function (proof, index, ar) {
+      return (!proof.meta || proof.meta === metaNone)
+    })
 
     const renderProps: RenderProps = {
       bioProps: {
         username: this.props.username,
         userInfo: this.props.userInfo,
-        currentlyFollowing: this.props.currentlyFollowing
+        currentlyFollowing: currentlyFollowing
       },
       headerProps: {
         reason: this.props.reason,
         onClose: () => this.props.onClose(this.props.username),
         trackerState: this.props.trackerState,
-        currentlyFollowing: this.props.currentlyFollowing
+        currentlyFollowing: currentlyFollowing,
+        changed: changed
```

Comment:
if the key and value are the same you can just use the var name. so like `{currentlyFollowing, changed}`